components/scream/src/physics/shoc/tests/shoc_run_and_cmp.cpp

```diff
@@ -93,26 +93,59 @@ static Int compare (const std::string& label, const Scalar* a,
 
 struct Baseline {
 
-  // Number of iterations (nadv steps of size dtime per iteration).
-  const int num_iters = 10;
-
-  Baseline () {
-    //                 ic, shcol, nlev, num_qtracers, nadv, dtime
-    params_.push_back({ic::Factory::standard, 8, 72, 3, 15, 150});
+  Baseline (const Int num_iters, const Int shcol, const Int repeat) :
```

Comment:
Was it in your intent to modify also src code in scream? Or did this spill from some unstaged work in your tree?

Reply:
@bartgol , this change was related to item 5 in the PR description. As I started to dive into improving the documentation of perf-analysis so that @tcclevenger can use it for measuring SHOC, I started trying to remember how I got it to work for measuring P3, starting poking around, and soon had shoc_run_and_cmp working.